File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float dollars;
    int Q, D, N, P, n, m, l;

    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars <= 0);

    int cents = round(dollars * 100);

    Q = cents / 25;
    n = cents % 25;

    D = n / 10;
    m = n % 10;

    N = m / 5;
    l = m % 5;

    P = l;

    printf("%i\n", Q + D + N + P);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    dollars = get_float("Change owed: ")
    if dollars > 0:
        coins = [25, 10, 5, 1]
        cents = round(dollars * 100)

        counter = 0

        for i in coins:
            counter += int(cents / i)
            cents = cents % i

        print(counter)
        break